docs/tasks/argo.rst

```diff
@@ -0,0 +1,253 @@
+You may read more about RBAC authorization here - RBAC_.
+
+1. Create a ClusterRole named ``cron-wf-manager`` having the permissions in
+   ``cluster-admin``.
+2. Grant these permissions to the default ServiceAccount named
+   ``default`` in the ``argo`` namespace.
+3. This will be done by creating a RoleBinding in the ``kanister``
+   and ``mysql-test`` namespaces.
+
+Execute the following -
+
+.. code-block:: bash
+
+  kubectl create rolebinding cron-wf-manager --clusterrole=cluster-admin --serviceaccount=argo:default -n kanister
+
+.. code-block:: bash
+
+  kubectl create rolebinding cron-wf-manager --clusterrole=cluster-admin --serviceaccount=argo:default -n mysql-test
+
+Step 5 - Launching the Cron Workflow
+````````````````````````````````````````````
+
+Lets launch the workflow in the ``argo`` namespace by running -
+
+.. code-block:: bash
+
+  argo cron create mysql-cron-wf.yaml -n argo
+
+Check if the workflow was created by running -
+
+.. code-block:: bash
+
+  argo cron list -n argo
+
+When the workflow runs, check if the ActionSet was created in the ``kanister`` namespace -
+
+.. code-block:: bash
+
+  kubectl get actionsets.cr.kanister.io -n kanister
+
+.. image:: img/argo-cron-created.png
+
+Here, the workflow was created and scheduled to run in 1 minute. After it ran
+successfully, the `last run` field was updated with the timestamp of the last run.
+A backup ActionSet was created. The age of this ActionSet is seen as recently created.
+
+Submit the Cron Workflow by running -
+
+.. code-block:: bash
+
+  argo submit mysql-cron-wf.yaml
+
+You should see the following on the Argo UI -
+
+.. image:: img/argo-cron-created-ui-list.png
+
+On clicking on it -
+
+.. image:: img/argo-cron-created-ui-desc.png
+
+Troubleshooting
+===============
+
+If the Cron Workflow does not run, check if the pod to run the workflow was
+created in the ``argo`` namespace. Examine the logs of this pod.
+
+.. code-block:: bash
+
+  kubectl logs [NAME_OF_MYSQL_CRON_WORKFLOW_POD] -n argo
+
+If this pod was not created, examine the logs of the Argo Workflow Controller
+in the ``argo`` namespace.
+
+.. code-block:: bash
+
+  kubectl logs [NAME_OF_WORKFLOW_CONTROLLER] -n argo
+
+If the logs mention that you have not granted the right permissions to the
+ServiceAccounts, circle back to Step 4 and check the names of your ServiceAccounts
+and namespaces.
+
+.. code-block:: bash
+
+  kubectl get serviceaccounts -n argo
+
+Cleanup
+===============
+
+Delete the cron workflow by running the following. Verify the name of your
+workflow before deleting it.
+
+Verify workflow name -
+
+.. code-block:: bash
+
+  argo cron list -n argo
+
+Delete workflow -
+
+.. code-block:: bash
+
+  argo cron delete mysql-cron-wf
+
+Deleting the Argo CRs.
+
+Run -
+
+.. code-block:: bash
+
+  kubectl delete -f quick-start-minimal.yaml
+
+Deleting the Argo namespace -
+
+.. code-block:: bash
+
+  kubectl delete namespace argo
+
+
```

Comment:
are we using the image `docs/tasks/img/argo-actionset-created.png` somewhere? I don't see that being used.

Reply:
Yupp. Removing it.